jwala-webapp/build.gradle

```diff
@@ -124,7 +124,7 @@ task watchJsx(type: Exec) {
 
 node {
     // Version of node to use.
-    version = '0.11.10'
+    version = '6.9.4'
```

Comment:
Just curious: can you explain why this stopped working if the version never changed before, and why do we have to jump from 0.11.10 all the way to 6.9.4?

Reply:
There seems to be an issue with npm Version compatibility and we could see an issue(https://github.com/cerner/jwala/issues/206) reported on the same in May 2018 by one of the Jwala user. We did a lot of trial and error and finally able to fix this up with Version 6.9.4. This version was mentioned as a solution in one of the forums too. Not sure exactly on what went wrong on the NPM side.!

Reply:
Sounds good.

For future reference, add comments like this to the commit message in Git so that the Git log will contain the justification for the version update.